indexing.rdf

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0"?>
+<?xml-stylesheet type="text/xsl" href="rdfs2html.xsl"?>
+<rdf:RDF 
+     xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#"
+     xmlns:xsd="http://www.w3.org/2001/XMLSchema#"
+     xmlns:dcterms="http://purl.org/dc/terms/"
+     xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#">
+
+    <rdf:Description rdf:about="http://fedora.info/definitions/v4/indexing#">
+      <dcterms:title xml:lang="en">Fedora Commons Indexing Ontology</dcterms:title>
+      <dcterms:modified rdf:datatype="http://www.w3.org/2001/XMLSchema#date">2015-01-23</dcterms:modified>
+      <dcterms:publisher rdf:resource="http://fedora.info"/>
+      <rdfs:seeAlso rdf:resource="https://wiki.duraspace.org/display/FEDORA4x/External+Search"/>
+      <rdfs:comment xml:lang="en">Ontology for indexing configuration in Fedora repositories.</rdfs:comment>
+    </rdf:Description>
+
+    <rdf:Description rdf:about="http://fedora.info/definitions/v4/indexing#hasIndexingTransformation">
```

Comment:
I had originally implemented this with rdf:Property, but @escowles XSLT expected rdf:Description (and I usually do everything possible to avoid writing XSLT). That said, I could also just take a deep breath and write some XSL.